tests/e2e/registry/test_registry_image_push_pull.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import pytest
+from ocs_ci.framework.testlib import tier1, E2ETest
+from ocs_ci.ocs import ocp, registry, constants
+from ocs_ci.framework import config
+
+logger = logging.getLogger(__name__)
+
+
+class TestRegistryImagePullPush(E2ETest):
+    """
+    Test to check Image push and pull worked with registry backed by OCS
+    """
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-1080")
+    def test_registry_image_pull_push(self):
+        """
+        Test case to validate registry image pull and push with OCS backend
+
+        """
+        image_url = 'docker.io/library/busybox'
+
+        # Get openshift registry route and certificate access
+        registry.enable_route_and_create_ca_for_registry_access()
+
+        # Add roles to user so that user can perform image pull and push to registry
+        registry.add_role_to_user(role_type='registry-viewer', user=config.RUN['username'])
```

Comment:
role_types = ['registry-viewer', 'registry-editor', 'system:registry', 'admin', 'system:image-builder']
for role_type in role_types:
    registry.add_role_to_user(role_type=role_type, user=config.RUN['username'])

Reply:
ack